modules/spacecraftinstruments/rendering/renderablemodelprojection.cpp

```diff
@@ -67,18 +70,44 @@ namespace {
         // This specifies the model that is rendered by the Renderable.
         std::filesystem::path geometryFile;
 
+        enum class [[codegen::map(openspace::DistanceUnit)]] ScaleUnit {
+            Nanometer,
+            Micrometer,
+            Millimeter,
+            Centimeter,
+            Decimeter,
+            Meter,
+            Kilometer,
+
+            // Weird units
```

Comment:
This subjective comment could be removed ;) Inches and feet are not weird to everyone

Reply:
The comment has lived long enough as an easter egg and it's fine to remove now :+1: